site/pages/about-us/q-and-a-slice/category/style.css

```diff
@@ -22,7 +22,8 @@
   width: 100%;
 }
 
-.category__questionList li:last-of-type h4 {
+.category__questionList li:last-of-type h4,
+.category__questionList li:last-of-type p {
   padding-bottom: 0;
```

Comment:
@gregdorward I've changed the way I've done the padding to something much simpler. Let me know if it works or not.